### ISSUE TITLE
Should we ask "power users" for information?

### ISSUE THREAD

**Issue by betatim** [jupyterhub/team-compass#72]
Inspired by issue an issue in the pangeo org: https://github.com/issues?q=440+user%3Apangeo-data

Should we ask people who regularly/once plan to use mybinder.org for a workshop/tutorial/course for a short note?

**Comment by choldgraf** [comment]
I think its a great idea - I think this should be done for JupyterHub as well (not just Binder)

**Comment by betatim** [comment]
How would we go about it though? Twitter campaign and then visiting some of the repositories that create bumps in the traffic?

I'd formulate it as "help us help you by giving us some information so we can show to our funders that their money is well spent which means this service will keep existing longer". WDYT?

**Comment by choldgraf** [comment]
yeah, some combination of internet spam + a few specific people we'd like to ask questions of would be great.

**Comment by choldgraf** [comment]
@betatim do you think there's a path forward on this? The "Binder survey" was a step in this direction, do you think we should do the same thing that's targeted at workshops/teaching? Or shall we close this and open up issues in the future as this topic comes up again?

**Comment by consideRatio** [comment]
Oooooooh I'd love to know more about the deployments out there, the more I've learned about who benefit from any work done on a project, the more I care to contribute to it. Z2JH was extremely motivating to me as it helped people in education and research for example, and even big universities!

**Comment by betatim** [comment]
Looking at the original comment for this issue I realised it doesn't link to where I thought it did.

It should have linked to https://github.com/pangeo-data/pangeo/issues/440

I think it would be good to have collecting like the pangeo issue for reporting to funding agencies, etc. It would also help us to not schedule upgrades or other explorative work when people have declared that they will do a workshop. It would probably also raise/create the expectation that we will do some kind of special thing for people's workshops (see [this](https://discourse.jupyter.org/t/binder-workshop-sun-july-21-9-am-to-1-pm-swiss-time-50-users/1731) and [this](https://discourse.jupyter.org/t/mp-workshop-on-mybinder/1714) which seem to at least between the lines ask if we would do something to make their workshop run well (the answer is "we don't have the resources to do so").

So maybe doing a retroactive survey which is as short/easy to answer as the "Binder user survey" we ran is the way forward. The "did you know" for it could be "if your workshop has less than 100 participants nothing special has to be arranged."  WDYT?
